src/Labels/Labels.php

```diff
@@ -423,4 +430,21 @@ private function getWorkableLabels(): array
 			'workableSuccess' => \__('Application submitted successfully. Thank you!', 'eightshift-forms'),
 		];
 	}
+
+	/**
+	 * Return labels - Jira
+	 *
+	 * @return array<string, string>
+	 */
+	private function getJiraLabels(): array
+	{
+		return [
+			'jiraMissingProject' => \__('Your form is missing project key. Please try again.', 'eightshift-forms'),
+			'jiraMissingIssueType' => \__('Your form is missing issue type. Please try again.', 'eightshift-forms'),
+			'jiraMissingSummary' => \__('Your form is missing issue summary. Please try again.', 'eightshift-forms'),
+			'jiraMissingEpicName' => \__('Your form is missing epic name key. Please try again.', 'eightshift-forms'),
+			'jiraBadRequestError' => \__('Something is not right with the job application. Please check all the fields and try again.', 'eightshift-forms'),
+			'jiraSuccess' => \__('Application submitted successfully. Thank you!', 'eightshift-forms'),
```

Comment:
```suggestion
			'jiraMissingProject' => \__('Project not set, please try again.', 'eightshift-forms'),
			'jiraMissingIssueType' => \__('Issue type not set, please try again.', 'eightshift-forms'),
			'jiraMissingSummary' => \__('Issue summary not set, please try again.', 'eightshift-forms'),
			'jiraMissingEpicName' => \__('Epic name not set, please try again.', 'eightshift-forms'),
			'jiraBadRequestError' => \__('The issue request is not valid. Please check all the fields and try again.', 'eightshift-forms'),
			'jiraSuccess' => \__('Issue submitted successfully!', 'eightshift-forms'),
```

Reply:
this should be labels for output to the user this over simplified labels are not the way to go